extensions/braze/braze-currents-users-event.example.1.json

```diff
@@ -0,0 +1,25 @@
+{
+  "braze:id": "a1234567-89ab-cdef-0123-456789abcdef",
+  "braze:time": 1477502783,
+  "braze:timezone": "America/Chicago",
```

Comment:
- AEP cannot process the timezone; it can just carry forward it as a data field.